store/tikv/coprocessor.go

```diff
@@ -570,8 +570,8 @@ func (it *copIterator) open(ctx context.Context) {
 		sendRate: it.sendRate,
 	}
 	taskSender.respChan = it.respChan
-	go taskSender.run()
 	it.actionOnExceed.setEnabled(true)
+	go taskSender.run()
 }
```

Comment:
We should enable action before sending task, otherwise the worker may meet OOM Action when rateLimitAction haven't been enabled.